src/utils/getExpirationDateFormat.ts

```diff
@@ -0,0 +1,5 @@
+import { parse } from "date-fns";
+
+export const getExpirationDateFormat = (date: string) => {
+  return parse(date, "dd.MM.yyyy HH:mm", new Date());
```

Comment:
As we discussed before it's better to have some general const for date format because you use it a few times - (setExpirationDateFormat, TodoDateFormat)